lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -28,6 +30,18 @@
                 render: function() {
                     var filledTemplate = this.tpl(this.model.toJSON());
                     HtmlUtils.setHtml(this.$el, filledTemplate);
+                    this.postRender();
+                },
+
+                postRender: function(){
+                    HtmlUtils.setHtml(this.$el.find('.course-actions'),
+                        HtmlUtils.HTML(
+                            new CourseCardActions({
+                                model: this.model,
+                                context: this.context
+                            }).$el.html()
+                        )
+                    );
```

Comment:
I got it working. The problem is `$el` cannot be auto assigned through the constructor. It has to be assigned again.